### DISCUSSION TITLE
how is update rate determined?

### DISCUSSION THREAD

**Discussion by garbb** [FreeRDP/FreeRDP#8189] [Q&A]
Does anyone know how the update rate is determined (that is, the rate at which screen changes/images of the screen are sent from the server to the client)? Is this something that is determined entirely by the server upon connection? Is there anything that can be changed within the client code to change the update rate?
The reason I am asking is that I am interested in trying to increase the update rate in order to improve performance on a high speed LAN network. Any information about where in the code I can look to possibly change the update rate would be helpful.
Thanks

**Comment by akallabeth** [comment]
1. updates are controlled by the server
2. there is a suspend/resume pdu to stop/resume screen updates (e.g. used when minimzing a window)
